etcd/v1alpha1/types_pacemakerstatus.go

```diff
@@ -0,0 +1,312 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// QuorumStatusType represents the quorum status of a Pacemaker cluster
+// +kubebuilder:validation:Enum=Quorate;NoQuorum
```

Comment:
I think we need to document the valid values, like:

```suggestion
// Valid values are Quorate (cluster has quorum) and NoQuorum (cluster does not have quorum).
// +kubebuilder:validation:Enum=Quorate;NoQuorum
```

The same applies to `NodeOnlineStatusType`, `NodeModeType`, `ResourceActiveStatusType` below.